.github/actions/create-jira-issues-for-community-activities/Initialize-IssueDetails.ps1

```diff
@@ -0,0 +1,50 @@
+param ($GitHub)
+
+switch ($GitHub.event_name)
+{
+    "discussion"
+    {
+        $summary = "Respond to `"$($GitHub.event.discussion.title)`""
+        $description = $Env:DISCUSSION_JIRA_ISSUE_DESCRIPTION
```

Comment:
I wanted to minimize passing around secrets.

Reply:
You could still pass those environment variables as parameters in the action where this script is called. Just to avoid depending on hidden variables.